configs/x86_64.config

```diff
@@ -0,0 +1,20 @@
+CT_CONFIG_VERSION="2"
+CT_LOCAL_TARBALLS_DIR="${CT_PREFIX:-${HOME}/x-tools}/sources"
+# CT_PREFIX_DIR_RO is not set
+CT_PATCH_BUNDLED_LOCAL=y
+CT_LOCAL_PATCH_DIR="${CT_TOP_DIR}/../../patches"
+CT_ARCH_X86=y
+CT_MULTILIB=y
+CT_ARCH_64=y
+CT_TARGET_VENDOR="multilib"
```

Comment:
Why call this `multilib` instead of `zephyr`?

Reply:
I think that *-zephyr-* implies that we are using newlib. However, this is using glibc instead because newlib does not fully support x32. It would result in linking errors for zephyr.elf.

Reply:
I don't see how or why `-zephyr-` would imply newlib.  I think it just means its from our SDK.

Reply:
Ok. I will change the tuple then.

Reply:
Changed vendor to zephyr.